src/charm.py

```diff
@@ -239,16 +239,27 @@ def _loki_config(self) -> dict:
                 kvstore:
                   store: inmemory
 
+            storage_config:
+              boltdb:
+                directory: /loki/boltdb-shipper-active
```

Comment:
A previous commit changed `boltdb-shipper` to `boltdb`.
I wonder why the path has `shipper` in it?

Reply:
Because of the volume mount in `metadata.yaml`